docs/design/operator.md

```diff
@@ -272,6 +275,9 @@ spec:
       app: cephfs-plugin
     annotations:
       k8s.v1.cni.cncf.io/networks: macvlan-conf-1
+    logRotator:
+      cpu: "500m"
```

Comment:
These seem like high resources for infrequent log rotation. Could you take a closer look at the memory usage of the log rotator sidecar? And probably the cpu can be easily reduced to `100m`.

Reply:
```
[rider@localhost examples]$ kubectl exec -it csi-rbdplugin-provisioner-58b644bdcc-mcxb9  -nrook-ceph -c log-collector sh
kubectl exec [POD] [COMMAND] is DEPRECATED and will be removed in a future version. Use kubectl exec [POD] -- [COMMAND] instead.
sh-4.4# cat /sys/fs/cgroup/cpu/cpuacct.usage
193634392
sh-4.4# cat /sys/fs/cgroup/memory/memory.usage_in_bytes
4141056
sh-4.4# 
sh-4.4# 

```

CPU: 193.6 M
Memory: 4M

Reply:
Currently there is no much logs,

SO I can put 200M to both

Reply:
What are the units for the cpuacct.usage? I don't understand how you arrived at `cpu: 200m`.
And if there is only 4M memory usage, why set `memory: 256Mi`? We could set it much lower such as `memory: 32Mi`.

Reply:
What are the units for the cpuacct.usage? 
bytes

Okay can set to 32Mi, 
container memory here is processing memory or the actually storage value for log files?

Reply:
> What are the units for the cpuacct.usage? 
> bytes

Ok I'd still suggest to set the cpu to 100m, I don't see the need for 200m.

> Okay can set to 32Mi, 
> container memory here is processing memory or the actually storage value for log files?

Yes it's just memory, not storage space.

Reply:
@travisn have we got to a resolution on this?

Reply:
I believe I have changed the values as per the suggestion, but we can wait for the reply

Reply:
i can see `100m` is added as requested, marking this as done on behalf of @travisn (as he is on PTO)